docs/customresourcestate-metrics.md

```diff
@@ -385,3 +385,35 @@ Examples:
 # if the value to be matched is a number or boolean, the value is compared as a number or boolean  
 [status, conditions, "[value=66]", name]  # status.conditions[1].name = "b"
 ```
+
+### [Variable VKs](https://github.com/kubernetes/kube-state-metrics/pull/1851)
+
+The CRS configuration allows you to monitor all versions and/or kinds that come under a group. 
+Taking the `Foo` object as reference the configuration below allows you to monitor all objects under all versions that come under the `myteam.io` group.
+
+```yaml
+kind: CustomResourceStateMetrics
+spec:
+  resources:
+    - groupVersionKind:
+        group: "myteam.io"
+        version: "*" # Set to `v1 to monitor all kinds under `myteam.io/v1`.
+        kind: "*" # TODO: Set to `Foo` to monitor all `Foo` objects under the `myteam.io` group (under all versions).
+      metrics:
+        - name: "foo_metrics"
+          help: "Foo Bar Baz"
+          each:
+            type: Info
+            info:
+              path: [metadata]
+              labelsFromPath:
+                object: [name]
+                namespace: [namespace]
+```
+
+The configuration above produces these metrics.
+
+```yaml
+kube_customresource_foo_metrics_myteam.io_v1_Foo{customresource_group="myteam.io",customresource_kind="Foo",customresource_version="v1",namespace="ns",object="foo"} 1
```

Comment:
I really would prefer for the metric names to not be dynamic. It makes it really hard to do any aggregation across CRD metrics...

Reply:
Yeah sorry about that, not sure what I was thinking, fixed.